notebooks/vertex_genai/solutions/gemini_function_calling.ipynb

```diff
@@ -0,0 +1,1369 @@
+{
```

Comment:
<p>I'd add a cell below which calls bq directly and check if the number is correct.</p>

<pre class="ql-syntax" spellcheck="false">%%bigquery
SELECT store_name, SUM(bottles_sold) AS total_bottles_sold FROM <code>kylesteckler-sandbox.iowa_liquor_sales.sales</code> GROUP BY store_name ORDER BY total_bottles_sold DESC LIMIT 1
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/493/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>